modules/osdk-upgrading-v1-7-1.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-upgrading-projects.adoc
+
+:osdk_ver: v1.8.0
+:osdk_ver_n1: v1.3.0
+:upgrade_ver: v1.7.1
+:upgrade_ver_n1: v1.6.1
+
+[id="osdk-upgrading-v1-7-1_{context}"]
+= Upgrading projects for Operator SDK {upgrade_ver}
+
+The following upgrade steps are associated with Operator SDK {upgrade_ver} and must be performed before completing an upgrade of an Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+- Operator SDK {osdk_ver} installed
+- Operator project that was previously created or maintained with Operator SDK {osdk_ver_n1}
+- Upgrade steps already performed for Operator SDK {upgrade_ver_n1}
+
+.Procedure
+
+. Optional: Add a manager config patch to the `config/default/kustomization.yaml` file.
++
+The generated `--config` flag was not added to either the `ansible-operator` or `helm-operator` binary when link:https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html[config file] support was originally added, so it does not currently work. The `--config` flag supports configuration of both binaries by file; this method of configuration only applies to the underlying link:https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager[controller manager] and not the Operator as a whole.
++
+To optionally configure the Operator's deployment with a config file, make changes to the `config/default/kustomization.yaml` file as shown in the following diff:
++
+.`config/default/kustomization.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+# If you want your controller-manager to expose the /metrics # endpoint w/o any authn/z, please comment the following line.
+\- manager_auth_proxy_patch.yaml
++# Mount the controller config file for loading manager configurations
++# through a ComponentConfig type
++- manager_config_patch.yaml
+----
+====
++
+This feature is optional; flags can be used as is or to override config file values.
+
+. Deprecate support for package manifests.
++
+The link:https://operatorframework.io/[Operator Framework] is removing support for the Operator package manifest format in a future release. As part of the ongoing deprecation process, the `operator-sdk generate packagemanifests` and `operator-sdk run packagemanifests` commands are now removed. To migrate package manifests to bundles, the `operator-sdk pkgman-to-bundle` command can be used.
```

Comment:
The commands haven't been removed yet, only deprecated for now.
```suggestion
The link:https://operatorframework.io/[Operator Framework] is removing support for the Operator package manifest format in a future release. As part of the ongoing deprecation process, the `operator-sdk generate packagemanifests` and `operator-sdk run packagemanifests` commands are deprecated. To migrate package manifests to bundles, the `operator-sdk pkgman-to-bundle` command can be used.
```

If this was written intentionally, then please do ignore this comment.

Reply:
Oh OK, I had seen locally that they were not listed in `--help` output anymore (and from the QE tests in https://issues.redhat.com/browse/OSDK-1622), but I guess they're technically still there?

Reply:
yeah, its still there. Since its deprecated and no longer supported, it doesn't appear in the help text.